pydantic_models/Treatment.py

```diff
@@ -0,0 +1,190 @@
+        examples="TR-7391493")
+    
+    eventId: str = Field(
+        description="Foreign key to map a patient to the event", 
+        examples="EV-20201123")
+
+    treatmentDate: date = Field(
+        description="Date of treatment", 
+        examples="2020-11-04")
+    
+    treatmentName: str = Field(
+        description="Name of the treatment given to the samples i.e. name of the chemical/drug/therapy", 
+        examples="Carboplatin")
+    
+    treatmentResponse: Literal['Partial Response', 'Complete Response', 'Stable Disease', 'Progressive Disease', 'No Response'] = Field(
+        description="This field indicates the type/extent of response to the treatment", 
+        examples="Partial Response")
+    
+    treatmentType: Literal['Medication','Chemotherapy', 'Radiation Therapy', 'Surgery', 'Cognitive behavioral therapy'] = Field(
+        description="Medication,Chemotherapy,Radiation Therapy,Surgery on Primary Tumor, Surgery on Lymph Node, cognitive behavioral therapy", 
+        examples="Chemotherapy")
+    
+    treatmentTypeDetail: Literal['Biopsy', 'Resection, 3D conformal', 'Intensity-modulated radiation therapy', 'Volumetric modulated radiation therapy', 'Image-guided radiation therapy', 'Stereotactic radiosurgery', 'Brachytherapy',' Superficial x-ray radiation therapy', 'Intraoperative radiation therapy', 'Other', 'Unknown'] = Field(
+        description="Treatment details which capture type of surgery, types of radiotherapy",
+        examples="Biopsy")
+
+    treatmentDosage: float = Field(
+        description="Treatment dosage for the drug administered over a defined period. Dosages can be increase or decreased as disease/condition improves or degrades", 
+        examples="100",
+        ge=0)
+    
+    treatmentAnatomicSite: str = Field(
+        description="Anatomical site of administered drug or chemotherapy",
+        examples="arm"
+    )
+
+    treatmentAnatomicalSiteType: Literal["Primary Tumor", "Lymph Node", "Metastatic"] = Field(
+        description="Anatomical site type such as primary, lymph node or metastatic",
+        examples="Metastatic"
+    )
+
+    daysPerCycleOfChemotherapy: int = Field(
+        description="Days per cycle of chemotherapy",
+    )
+
+    numberOfCyclesOfChemotherapy: int = Field(
+        description="The number of cycles of chemotherapy refers to the repeated administration of chemotherapy drugs over a defined period",
+        examples="4",
+        ge=1
+    )
+
+    routeOfAdministration: Literal["Oral", "Intravenous", "Subcutaneous"] = Field(
+        description="The route of administration describes how chemotherapy drugs are delivered into the body",
+        examples="Intravenous"
+    )
+
+    adverseEventList: str = Field(
+        description="Adverse events refer to undesirable or harmful effects experienced by patients as a result of chemotherapy or drug treatments",
+        examples="Nausea"
+    )
+
+    adverseEventTreatment: str = Field(
```

Comment:
Can there be general validation that treatmentType IS NOT Unknown or None while adding values to all fields?